### DISCUSSION TITLE
[Feature request]: MacOS client one day?

### DISCUSSION THREAD

**Discussion by BitGrub** [ClassicOldSong/moonlight-android#57] [General]
### Is your feature request related to a problem? Please describe.

Hey, I just discovered your Sunshine fork, Apollo, and had to say thank you, and congratulations, it's damn-near perfect.
Anyway, I see iOS and Desktop clients are planned in the future, just wondering if MacOS is included in one of them. Hope it is!

### Describe the solution you'd like

Moonlight works pretty well, and other than a slightly easier pairing process I don't think I'm missing much. Still, would love to see a fork for MacOS as well one day

### Describe alternatives you've considered

Moonlight

**Comment by ClassicOldSong** [comment]
All desktop client share one code base so it'll be included.
